hooks/utils.py

```diff
@@ -132,6 +132,7 @@ class FormatterCmd(Command):
     def __init__(self, command, look_behind, args):
         super().__init__(command, look_behind, args)
         self.file_flag = None
+        self.show_diff = True
```

Comment:
Change name of var and to `None` as well.

Reply:
Done in 652bda6